react-components/src/components/RuleBasedOutputs/core/checkStringExpressionStatement.test.ts

```diff
@@ -0,0 +1,98 @@
+/*!
+ * Copyright 2025 Cognite AS
+ */
+import { describe, it, expect } from 'vitest';
+import { type FdmTyping, type StringExpression, type TriggerTypeData } from '../types';
+import { type Asset } from '@cognite/sdk';
+import { checkStringExpressionStatement } from './checkStringExpressionStatement';
+
+describe('checkStringExpressionStatement', () => {
+  const triggerTypeData: TriggerTypeData[] = [
+    {
+      type: 'metadata',
+      asset: {
+        metadata: {
+          key1: 'value1',
+          key2: 'value2'
+        }
+      } as unknown as Asset
```

Comment:
Another unknown-cast - but in this case I understand that it would be annoying to add all the correct fields :sweat_smile: